github/commitstatus.go

```diff
@@ -198,11 +198,14 @@ func (cs CommitStatus) explainError(err error, state, sha, url string) error {
 // Example:
 // if hostname is: github.com it returns https://api.github.com
 // if hostname is overrriden in tests by localhost HTTP testserver it returns http://127.0.0.1:PORT
+// if hostname is Github Enterprise instance (e.g., github.mycompany.org) it returns e.g., https://github.mycompany.org/api/v3
```

Comment:
// otherwise, hostname is assumed to be of a Github Enterprise instance. For example, github.mycompany.org returns https://github.mycompany.org/api/v3